.github/workflows/build.yaml

```diff
@@ -60,8 +60,7 @@ jobs:
     # while Python operates within the runner's global environment. It is safe to ignore
     # warnings from the Makefile about the missing virtual environment.
     - name: Create empty virtual environment for Actions
-      run: |
-        mkdir .venv
+      run: mkdir .venv
```

Comment:
This is already addressed [here](https://github.com/jenstroeger/python-package-template/pull/291).

Reply:
Resolved after rebase.